src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -2620,16 +2620,32 @@ static void layoutGlissando(const Glissando* item, LayoutContext& ctx, Glissando
     offs2 *= -1.0;
     // Look at chord shapes (but don't consider lyrics)
     Shape cr1shape = cr1->shape();
-    mu::remove_if(cr1shape, [](ShapeElement& s) {
-        if (!s.toItem || s.toItem->isLyrics()) {
+
+    // don't consider symbols above and below noteheads
+    double yMin = cr1->upNote()->y() + cr1->upNote()->headHeight();
+    double yMax = cr1->downNote()->y();
+    mu::remove_if(cr1shape, [yMin, yMax](ShapeElement& s) {
+        if (!s.toItem || s.toItem->isLyrics()
+            || (s.toItem->isSymbol() && (s.toItem->y() < yMin || s.toItem->y() > yMax))) {
             return true;
         } else {
             return false;
         }
     });
+    Shape cr2shape = cr2->shape();
+    double yMin2 = cr2->upNote()->y() + cr2->upNote()->headHeight();
+    double yMax2 = cr2->downNote()->y();
+    mu::remove_if(cr2shape, [yMin2, yMax2](ShapeElement& s) {
+        if (!s.toItem || (s.toItem->isSymbol() && (s.toItem->y() < yMin2 || s.toItem->y() > yMax2))) {
+            return true;
+        } else {
+            return false;
+        }
+    });
```

Comment:
There is some code duplication here... perhaps it would be preferable to create a local lambda function and call that twice. I.e. 
```c++
    auto relevantShape(const Chord* cr) -> Shape {
        Shape s = cr->shape();
        double yMin = …; double yMax = …; …
        mu::remove_if(…);
        return s;
    }
    …
    offs1.rx() += relevantShape(cr1).right() - anchor1->pos().x();
```